index.html

```diff
@@ -242,7 +242,7 @@ <h3><span><img src="images/Rectangle 57.png" alt=""></span>&nbsp; Skills</h3>
                 </span>
                 <input  type="email" placeholder="Email address">
                 <textarea name="comments" id="" cols="30" rows="3" placeholder="Write me something..."></textarea>
-                <button>Get in touch</button>
+                <button class="contact-btn" type="button">Get in touch</button>
```

Comment:
- [ ] This is a button to submit the form inputs and therefore following the best practices, it should have a type of `submit`, I'll suggest you update this. :+1: